db/README.md

```diff
@@ -1,4 +1,40 @@
-### schema explanation
+### schema explanation (as of 2017-12-14)
+
+##### Schema Entity-Relationship Diagram
+![Schema Entity-Relationship Diagram as of 2017-12-14](schema_ER_diagram.2017-12-14.png)
+
+###### Key:
+* <sub>The diagram is generated from a Postgres schema dump, so the tables are listed by their SQL names.  Each SQL table has a corresponding ActiveRecord class, whose name is a camel case version of the snake case SQL name (with one exception, explained below).  The explanation tries to use the SQL table name or the ActiveRecord classname according to which is most appropriate, and may be arbitrary when either would suffice.</sub>
+* <sub>The cross side of the connection between tables indictates a required foreign key relationship, or what ActiveRecord would call `belongs_to` (with a `null: false` constraint on the column definition and a corresponding `presence: true` on the ActiveRecord class definition).  In our current data model, none of our foreign key reference fields may be null.  Each such ActiveRecord object must point to exactly one instance of its foreign key referent.</sub>
```

Comment:
Maybe a mention that 'FK' in the diagram corresponds to 'foreign key' somewhere, and maybe the same for 'PK' and 'primary key'? I wouldn't hold up a merge for it, though.